mumbleBot.py

```diff
@@ -332,7 +329,11 @@ def start_web_interface(addr, port):
 
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='Bot for playing radio stream on Mumble')
+    global __CONFIG
+    parser = argparse.ArgumentParser(description='Bot for playing music on Mumble')
```

Comment:
Changing `radio stream` to `music` has nothing to do with the submitted feature. Please commit in another PR. Thanks 😃

Reply:
Fixed